Web/starter_project_barbies/components/about/ProjectCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { ProjectCardProps } from '@/types/AboutTypes'
+import Image from 'next/image'
+
+const ProjectCard = ({ project, isEven }: ProjectCardProps) => {
+  return (
+    <div
+      className={`flex flex-wrap justify-around ${
```

Comment:
[nit] can you try this format:

```
className={{
  flex,
  flexWrap,
  justifyAround,
  flexRowReverse: isEven
}}
// not entirely sure if the keys should be in camelCase or kebab-case with quotes
```

it is an alternative approach that might be cleaner